skywalking-collector/skywalking-collector-cluster/src/main/java/com/a/eye/skywalking/collector/actor/AbstractClusterWorker.java

```diff
@@ -11,18 +11,45 @@
 import org.apache.logging.log4j.Logger;
 
 /**
+ * The <code>AbstractClusterWorker</code> should be implemented by any class whose instances
+ * are intended to provide receive remote message that message will transfer across different jvm
+ * which running in same server or another address server.
+ * <p>
+ * Usually the implemented class used to receive persistence data or aggregation the metric.
+ *
  * @author pengys5
+ * @since feature3.0
```

Comment:
`@since` should only target a version, and `feature3.0` is not a version number, clearly.